server/controllers/account.coffee

```diff
@@ -68,7 +68,8 @@ module.exports =
                     return cb null, errors
 
                 unless utils.checkPassword oldPassword, user.password
-                    errors.push localizationManager.t "current password incorrect"
+                    errors.push localizationManager.t "current password" +
+                                                                " incorrect"
```

Comment:
@aenario I'd prefer not to split translation keys. This is something we may want to grep to see if it is still used when updating the locale files. Maybe something like:

```coffee
msg = localizationManager.t "current password incorrect"
errors.push msg
```